composer.json

```diff
@@ -18,9 +18,7 @@
     "jakub-onderka/php-parallel-lint": "^1.0",
     "localheinz/composer-normalize": "~0.9.0",
     "phing/phing": "^2.16.0",
-    "spryker/kernel": "*",
     "squizlabs/php_codesniffer": "^3.3.1",
-    "spryker/code-sniffer": "dev-master",
```

Comment:
Code sniffer should always be OK
please check why tools wants to remove, and make sure that part is added (see renes tool for this).